BackToBasicsUIKit/BackToBasicsUIKit/Demo ViewControllers/Jimmy/JimmySecondViewController.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  JimmySecondViewController.swift
+//  BackToBasicsUIKit
+//
+//  Created by Jimmy Gutierrez on 5/23/19.
+//  Copyright © 2019 SwiftMiami. All rights reserved.
+//
+
+import UIKit
+
+protocol canRecieve {
+    func dataRecieved(data: String)
+}
+
+class JimmySecondViewController: UIViewController {
+    
+    var delegate : canRecieve?
+    
+    var data = ""
+    
+    @IBOutlet var label: UILabel!
+    @IBOutlet var textField: UITextField!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        label.text = data
+
+    }
+    
+    
```

Comment:
Please remove all these empty lines. It may sound OCD but being neat with your code is very important. You're sharing this codebase with a team, please clean up after yourself.